src/client/components/MainWindow/MainWindow.js

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react';
+import SearchBlock from '../SearchBlock/SearchBlock';
+import SingleMovie from '../SingleMovie/SingleMovie';
+import './MainWindow.scss';
+
+
+export default class MainWindow extends Component {
+
+    constructor(props) {
+        super(props);
+        this.makeFetch = this.makeFetch.bind(this);
+        this.state = {
+            searchBlock: true,
+            foundMovies: 0,
+            movies: []
+        };
+    }
+
+    makeFetch(searchString = '', searchBy = '', offset = '', limit = '') {
+        let callString = 'http://react-cdp-api.herokuapp.com/movies';
+        callString += `?search=${searchString}&searchBy=${searchBy}&offset=${offset}&limit=${limit}`;
+
+        /* eslint-disable no-undef */
+        fetch(callString)
+            .then(function (response) {
+                return response.json();
+            })
+            .then(myJson => {
+                this.setState({movies: myJson.data})
+                this.setState({foundMovies: myJson.total})
```

Comment:
Please combine this in the one setState call to prevent unnecessary rerendering.

this.setState({
  movies: ...,
  foundMovies: ...
})